app/controllers/tag_controller.rb

```diff
@@ -288,9 +288,8 @@ def create
           node.add_comment(subject: 'co-author',
                            uid: current_user.uid,
                            body: " @#{current_user.username} has marked #{tagname.split(':')[1]} as a co-author. ")
-
         end
-
+        
```

Comment:
And if possible let's revert this file since there aren't any real changes, no?

Reply:
Sure!!